pkg/cmd/cluster-policy-controller/policy_controller.go

```diff
@@ -12,11 +12,14 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/serviceability"
 
 	origincontrollers "github.com/openshift/cluster-policy-controller/pkg/cmd/controller"
 )
 
 func RunClusterPolicyController(ctx context.Context, controllerContext *controllercmd.ControllerContext) error {
+	serviceability.InitLogrusFromKlog()
```

Comment:
I am not sure if we still need this one but I have not found a usage for this.

Reply:
Yeah, looks like this is unused can be dropped.

Reply:
dropped